admin/pre-reg/static/js/pages/prereg-admin-page.js

```diff
@@ -0,0 +1,292 @@
+'use strict';
```

Comment:
This file should probably live in a app root static/js/pre-reg directory unless we foresee this app being made available separately from the admin module sometime in the future.